images/icon-cloud.svg

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
```

Comment:
@brendastorer I create my icons in Sketch. Before I export I make sure I have done https://www.sketchapp.com/support/documentation/13-preferences/. This helps a bit. That is usually where I stop.

What other data do you think I should strip?